Pages/Index.cshtml.cs

```diff
@@ -24,15 +24,9 @@ public IndexModel(ILogger<IndexModel> logger, UserManager<ApplicationUser> userM
 
         public void OnGet()
         {
-            /*var user = userManager.GetUserAsync(HttpContext.User).Result;
-            name = user.UserName;
-            if (user != null)
-            {
-                role = userManager.GetRolesAsync(user).Result[0];
-            }
-            //var user = userManager.GetUserName(HttpContext.User);
-            //var role = userManager.GetRolesAsync(user);
```

Comment:
Why was all of this commented out? Did your forget to uncomment it?

Reply:
Long ago this was useless code. So now I just deleted it